### PR TITLE
0.17: Pre-branch maintenance

### PR DESCRIPTION
**Pull request by DrahtBot**
Some trivial maintenance to avoid having to do it again after the 0.17 branch off.

(The scripts to do this are in `./contrib/`)

### CODE REVIEW

File: src/qt/intro.cpp [hunk 1]
```diff
@@ -23,7 +23,7 @@
 
 static const uint64_t GB_BYTES = 1000000000LL;
 /* Minimum free space (in GB) needed for data directory */
-static const uint64_t BLOCK_CHAIN_SIZE = 200;
+constexpr uint64_t BLOCK_CHAIN_SIZE = 220;
```
Comment (Empact) [inline src/qt/intro.cpp]: `static`?

Comment (luke-jr) [reply src/qt/intro.cpp]: Is it possible for a constexpr to be non-static?

**Comment by Sjors** [comment]
Can the copyright headers be a scripted diff?

Why do we have generated manpages in the repo? (maybe not the right time to address that)

**Comment by MarcoFalke** [comment]
@Sjors They can't since the script takes into account the current time to determine what to bump to. So you coulnd't verify the scripted diff without setting the time to the commit time.

**Comment by Sjors** [comment]
What if you could pass `YEAR=2018` into the script, or if the year is committed first?

**Comment by MarcoFalke** [comment]
@Sjors Jup, that'd work. Though, I feel we should do that in a separate pull request.

**Comment by Sjors** [comment]
@MarcoFalke that's fine, this is easy enough to review. Is this ready for review or are you planing to add more?

**Comment by MarcoFalke** [comment]
I am not aware of anything else that needs to be done pre-branch

**Comment by Sjors** [comment]
ACK 3fc2063

**Comment by kallewoof** [comment]
utACK 3fc2063

**Comment by achow101** [comment]
utACK 3fc20632a3ad30809356a58d2cf0ea4a4ad4cec3

**Comment by laanwj** [comment]
I guess the manpages need to be re-regenerated after #13872
utACK otherwise!

**Comment by MarcoFalke** [comment]
@laanwj They'd have to be bumped every time the version is bumped on the 0.17 branch anyway. So imo it can be done later to not invalidate the review here.

**Comment by laanwj** [comment]
OK I thought the idea was to generate these just before the branch, but otherwise is ok me too.

**Pull request merged**